### PR TITLE
Set variables "convert" and "archive" in TUI

### LINKED ISSUE #22: Set variables in TUI
Now:
**command name** and **app activity** hardcoding in adb.py:
```python
device.cli_command_run('convert', db)
...
device.start_app('.ui.activities.main.MainActivity')
```

Need:
Set **command name** and **app activity** in TUI

### PR DESCRIPTION
**Pull request by larnaa**
Close #22

### CODE REVIEW

File: src/bluecoins_cli/adb.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def pull_db(self, db: str) -> None:
             check=True,
         )
 
-    def cli_command_run(self, cli_command: str, db: str) -> None:
-        subprocess.run(f'bluecoins-cli {db}.fydb {cli_command}', shell=True, check=True)
+    def cli_command_run(self, cli_command: str, db: str, keys: str) -> None:
```
Comment (droserasprout) [inline src/bluecoins_cli/adb.py:47]: I suggest renaming keys -> options/arguments.

See [argparse tutorial](https://docs.python.org/3/howto/argparse.html) to learn the difference.

File: src/bluecoins_cli/adb.py [hunk 2]
```diff
@@ -62,10 +62,17 @@ def start_app(self, activity: str) -> None:
...
 
 
-device = Device.connect()
-device.stop_app()
-db = get_db_name()
-device.pull_db(db)
-device.cli_command_run('convert', db)
-device.push_db_root(db)
-device.start_app('.ui.activities.main.MainActivity')
+def execute_cli_command_with_adb(cli_command: str, activity: str, keys_value: str = '') -> None:
```
Comment (droserasprout) [inline src/bluecoins_cli/adb.py:65]: It's unobvious what's `activity` argument purpose (to start after the final sync is complete). Also, optional?

**Comment by larnaa** [comment]
@droserasprout It's working, but don't use it on the prod base.

File: pyproject.toml [hunk 3]
```diff
@@ -12,22 +12,20 @@ packages = [
...
 python = "^3.10"
-anyio = "^3.6.1"
-aiohttp = "^3.8.1"
-asyncclick = "^8.0.3"
 xdg = "^5.1.1"
 adbutils = "^1.0.8"
+click = "^8.1.3"
+requests = "^2.28.1"
 
 [tool.poetry.dev-dependencies]
```
Comment (droserasprout) [inline pyproject.toml:25]: Rename to `[tool.poetry.group.dev.dependencies]`

File: src/bluecoins_cli/adb.py [hunk 4]
```diff
@@ -44,8 +44,8 @@ def pull_db(self, db: str) -> None:
             check=True,
         )
 
-    def cli_command_run(self, cli_command: str, db: str) -> None:
```
Comment (droserasprout) [inline src/bluecoins_cli/adb.py:47]: Unused function

File: src/bluecoins_cli/cli.py [hunk 5]
```diff
@@ -19,29 +19,61 @@
 from bluecoins_cli.storage import BluecoinsStorage
 from bluecoins_cli.storage import Storage
 
+# from bluecoins_cli.sync_manager import SyncManager
+from bluecoins_cli.tui import run_tui
+
 logging.basicConfig(level=logging.DEBUG)
 
 
+# sync = SyncManager()
```
Comment (droserasprout) [inline src/bluecoins_cli/cli.py:28]: Cleanup please

File: src/bluecoins_cli/cli.py [hunk 6]
```diff
@@ -19,29 +19,61 @@
...
 
 @click.group()
+@click.pass_context
+def root(
+    ctx: click.Context,
+) -> None:
+    ...
+
+
+@root.group(help='Start CLI')
```
Comment (droserasprout) [inline src/bluecoins_cli/cli.py:44]: Maybe something like "Commands to manage database lalala"?

File: src/bluecoins_cli/cli.py [hunk 7]
```diff
@@ -19,29 +19,61 @@
...
+
+
 @cli.command(help='Convert database to another main currency')
 @click.argument('base_currency', default='USD')
 @click.pass_context
-async def convert(
+def convert(
     ctx: click.Context,
     base_currency: str,
+    db: str,
```
Comment (droserasprout) [inline src/bluecoins_cli/cli.py:66]: No such option in click decorators.

File: src/bluecoins_cli/sync_manager.py [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+from bluecoins_cli.adb import generate_new_db_name
+
+device = Device.connect()
+
+
+class SyncManager:
+    def __init__(self) -> None:
+        self.db = generate_new_db_name()
+
+    """
```
Comment (droserasprout) [inline src/bluecoins_cli/sync_manager.py:11]: Docstrings must be placed right after class/def

File: src/bluecoins_cli/sync_manager.py [hunk 9]
```diff
@@ -0,0 +1,26 @@
+from bluecoins_cli.adb import Device
+from bluecoins_cli.adb import generate_new_db_name
+
+device = Device.connect()
```
Comment (droserasprout) [inline src/bluecoins_cli/sync_manager.py:4]: A side effect, unexpected actions on module import

File: src/bluecoins_cli/tui.py [hunk 10]
```diff
@@ -30,35 +32,92 @@
...
-            "",
-            [
-                "Convert",
-                lambda *_: call(
-                    ["python", ADB_SYNC_SCRIPT_PATH],
-                ),
-            ],
-            width=60,
-            box="DOUBLE",
+sync = SyncManager()
```
Comment (droserasprout) [inline src/bluecoins_cli/tui.py:35]: Side effect, see comment in sync_manager module.

File: src/bluecoins_cli/tui.py [hunk 11]
```diff
@@ -30,35 +32,92 @@
...
+        + ""
+        + Button('Back', lambda *_: manager.remove(window))
+    ).center()
+
+    return currency_window
+
+
+def get_choose_account_archive_window(manager: WindowManager) -> Window:
+    # FIXME: hardcode
+    path_test = '/home/larnaa/VScode_project/bluecoins-cli/Bluecoins_last_3.fydb'
```
Comment (droserasprout) [inline src/bluecoins_cli/tui.py:93]: I don't think we should merge it. Hardcode is okay for now, but not machine-specific.

**Review by droserasprout — CHANGES_REQUESTED** [review]

**Review by droserasprout — APPROVED** [review]

**Pull request merged**